packages/booter-lb3app/src/__tests__/acceptance/booter-lb3app.acceptance.ts

```diff
@@ -242,4 +242,20 @@ describe('booter-lb3app', () => {
       expect(ds).to.eql(expected);
     });
   });
+
+  context('binding LoopBack 3 models', () => {
+    before(async () => {
+      ({app, client} = await setupApplication({
+        lb3app: {path: '../fixtures/app-with-model'},
+      }));
+    });
+
+    it('binds model to the context', async () => {
+      const expected = require('../../../fixtures/app-with-model').models.Color;
+      const modelBindings = app.findByTag('model');
```

Comment:
Did this test pass CI? I would expect the tag to be `lb3-model`. 🤔 

```suggestion
      const modelBindings = app.findByTag('lb3-model');
```